project/Dependencies.scala

```diff
@@ -42,7 +42,7 @@ object Dependencies {
 
   val asyncHttpClient = Seq("org.asynchttpclient" % "async-http-client" % "2.12.3")
 
-  val akkaStreams = Seq("com.typesafe.akka" %% "akka-stream" % "2.6.20")
+  val pekkoStreams = Seq("org.apache.pekko" %% "pekko-stream" % "0.0.0+26669-ec5b6764-SNAPSHOT")
```

Comment:
The 1.0.1 relased.

Reply:
Yes, update this PR and published as snapshot: https://oss.sonatype.org/content/repositories/snapshots/com/typesafe/play/play-ws-standalone_2.13/2.2.0-M4+28-aac7b468-SNAPSHOT/ which depends on Pekko 1.0.1